posthog/models/property/util.py

```diff
@@ -220,7 +220,7 @@ def parse_prop_clauses(
                 idx,
                 prepend=f"personquery_{prepend}",
                 allow_denormalized_props=True,
-                transform_expression=lambda column_name: f"argMax(person.{column_name}, _timestamp)",
+                transform_expression=lambda column_name: f"argMax(person.{column_name}, version)",
```

Comment:
oof, not sure about this change.

For example, see this:

```
SELECT version, _timestamp, * from person where team_id = 2 and id = '01818d74-703c-0000-c7a3-f8602026dea4'
```

`version` and `_timestamp` reverse the rows.

Reply:
Don't know the background here, and which one is the source of truth

Reply:
I changed it back. There's no need to adjust this PR